marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/entities/search/EntitySearchManagerTest.java

```diff
@@ -96,7 +96,7 @@ public void testSearchResultsOnNonExistentEntityModel() {
 
         SearchQuery searchQuery = new SearchQuery();
         searchQuery.getQuery().setEntityTypeIds(Arrays.asList("Some-entityType"));
-        assertThrows(MarkLogicServerException.class, () -> new EntitySearchManager(getHubClient()).search(searchQuery), "Entity Model with name Some-entityType doesn't exist ");
+        assertNull(new EntitySearchManager(getHubClient()).search(searchQuery), "Entity Model with name Some-entityType doesn't exist ");
```

Comment:
@rahulvudutala ,
In entity-search-lib.sjs line 48, I changed the HTTP error code from 500 to 404. It causes this endpoint to return null instead of exception it used to throw earlier ? let me know if that's ok.

Reply:
Yeah it should be ok.